index.html

```diff
@@ -0,0 +1,163 @@
+            <div class="middle">
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fas fa-external-link-square-alt"></i> Show Live</a
+              ><br />
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fab fa-github-square"></i> GitHub</a
+              >
+            </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+          </section>
+          <section class="project">
+            <h3>Project 3</h3>
+            <img src="images/pexels-digital-buggu-374559.jpg" alt="Project 3" />
+            <div class="middle">
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fas fa-external-link-square-alt"></i> Show Live</a
+              ><br />
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fab fa-github-square"></i> GitHub</a
+              >
+            </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+          </section>
+          <section class="project">
+            <h3>Project 4</h3>
+            <img src="images/pexels-digital-buggu-374559.jpg" alt="Project 4" />
+            <div class="middle">
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fas fa-external-link-square-alt"></i> Show Live</a
+              ><br />
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fab fa-github-square"></i> GitHub</a
+              >
+            </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+          </section>
+          <section class="project">
+            <h3>Project 5</h3>
+            <img src="images/pexels-digital-buggu-374559.jpg" alt="Project 5" />
+            <div class="middle">
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fas fa-external-link-square-alt"></i> Show Live</a
+              ><br />
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fab fa-github-square"></i> GitHub</a
+              >
+            </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+          </section>
+          <section class="project">
+            <h3>Project 6</h3>
+            <img src="images/pexels-digital-buggu-374559.jpg" alt="Project 6" />
+            <div class="middle">
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fas fa-external-link-square-alt"></i> Show Live</a
+              ><br />
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fab fa-github-square"></i> GitHub</a
+              >
+            </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+          </section>
+        </div>
+        <div class="back-to-top">
```

Comment:
Is there a better option than `<div>`? (for accessibility)

Reply:
Changed it to p tag